scripts/setup.mongodb

```diff
@@ -10,15 +33,65 @@ try {
 ["ims", "test-ims"].forEach((databaseName) => {
   db = db.getSiblingDB(databaseName);
 
+  // Add some usage statuses if the collection doesnt already exist
+  if (!db.getCollectionNames().includes("usage_statuses")) {
+    console.log(`Populating usage statuses in ${databaseName}...`);
+
+    db.usage_statuses.insertMany([
+      createUsageStatusIn(USAGE_STATUS_ID_NEW, "New"),
+      createUsageStatusIn(USAGE_STATUS_ID_IN_USE, "In Use"),
+      createUsageStatusIn(USAGE_STATUS_ID_USED, "Used"),
+      createUsageStatusIn(USAGE_STATUS_ID_SCRAPPED, "Scrapped"),
+    ]);
+  }
+
   // Add some system types if the collection doesn't already exist
   if (!db.getCollectionNames().includes("system_types")) {
     console.log(`Populating system types in ${databaseName}...`);
 
     db.system_types.insertMany([
-      { _id: ObjectId("685e5dce6e347e39d459c5ea"), value: "Storage" },
-      { _id: ObjectId("685e5dce6e347e39d459c5eb"), value: "Operational" },
-      { _id: ObjectId("685d52f4b1095c1837c69dac"), value: "Quarantine" },
-      { _id: ObjectId("685d52f4b1095c1837c69dad"), value: "Scrapped" },
+      { _id: SYSTEM_TYPE_ID_STORAGE, value: "Storage" },
+      { _id: SYSTEM_TYPE_ID_OPERATIONAL, value: "Operational" },
+      { _id: SYSTEM_TYPE_ID_QUARANTINE, value: "Quarantine" },
+      { _id: SYSTEM_TYPE_ID_SCRAPPED, value: "Scrapped" },
+    ]);
+  }
+
+  // Add some rules if the collection doesn't already exist
+  if (!db.getCollectionNames().includes("rules")) {
+    console.log(`Populating rules in ${databaseName}...`);
+
+    db.rules.insertMany([
+      {
+        // Creation of items in storage -> New
+        src_system_type_id: null,
+        dst_system_type_id: SYSTEM_TYPE_ID_STORAGE,
+        usage_status_id: USAGE_STATUS_ID_NEW,
```

Comment:
```suggestion
        dst_usage_status_id: USAGE_STATUS_ID_NEW,
```
Might be clearer that it relates to the value that it will be at the destination. Any thoughts on the name of these fields?